scripts/gulp/create-bundle.js

```diff
@@ -117,6 +118,12 @@ module.exports = function createBundle(config, buildOpts) {
 
   var bundle = browserify([], bundleOpts);
 
+  bundle.transform(stringify, {
```

Comment:
All our other browserify transforms are defined in `package.json`. Should this not go there, too?